cli/src/org/partiql/cli/main.kt

```diff
@@ -93,6 +97,13 @@ private val inputFileOpt = optParser.acceptsAll(listOf("input", "i"), "input fil
     .withRequiredArg()
     .ofType(File::class.java)
 
+private val inputFormatOpt = optParser.acceptsAll(listOf("input-format", "if"), "output format, requires the query option")
```

Comment:
Might have missed this from the first review. Hint is incorrect

```suggestion
private val inputFormatOpt = optParser.acceptsAll(listOf("input-format", "if"), "input format, requires the query option")
```